### DISCUSSION TITLE
zh/docs/internationalization/

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#3195] [Docs comments]
# zh/docs/internationalization/

帮助casdoor支持不同语言

https://casdoor.org/zh/docs/internationalization/

**Comment by yhgroup** [comment]
antd带的国际化没有生效，很多

**Reply by yhgroup** [reply]
node_modules/antd/locale/

**Reply by hsluoyz** [reply]
> antd带的国际化没有生效，很多

@yhgroup create bug report issue in English
